### PR TITLE
Fix validate utility network message

### PR DESCRIPTION
**Pull request by har13205**
# Description

This PR is to fix the message displayed while validating utility network topology

## Type of change

- [x] Bug fix
- [ ] New sample implementation
- [ ] Sample viewer enhancement
- [ ] Other enhancement

**Platforms tested on**:

- [ ] Windows
- [ ] Android
- [ ] Linux
- [x] macOS
- [ ] iOS

### CODE REVIEW

**Review by ldanzinger — APPROVED** [review]

**Comment by GuillaumeBelz** [comment]
Off topic: I have a question about the `job` object. How this object is deleted? He has `UtilityNetwork` as parent, then it's not a leak, but I guess it should be deleted in the lambda.

File: ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.cpp [hunk 1]
```diff
@@ -461,7 +461,7 @@ void ValidateUtilityNetworkTopology::onValidate()
         m_busy = false;
         emit isBusy();
       }
-      else
+      else if(status == JobStatus::Failed)
```
Comment (GuillaumeBelz) [inline ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.cpp:464]: With `if` here, that means other enum values are not handled. I guess it's fine for `NotStarted`, `Started`, and `Paused`, but maybe the value `Canceling` should be handled.

Comment (har13205) [reply ArcGISRuntimeSDKQt_CppSamples/UtilityNetwork/ValidateUtilityNetworkTopology/ValidateUtilityNetworkTopology.cpp]: Thanks @GuillaumeBelz , I have made the changes as per your suggestion

**Review by GuillaumeBelz — APPROVED** [review]
Looks good. Thanks

**Pull request merged**